orderer/consensus/etcdraft/util.go

```diff
@@ -247,7 +247,7 @@ func VerifyConfigMetadata(metadata *etcdraft.ConfigMetadata, verifyOpts x509.Ver
 			return errors.Errorf("metadata has nil consenter")
 		}
 		if err := validateConsenterTLSCerts(consenter, verifyOpts, true); err != nil {
-			return err
+			return errors.Wrapf(err, "consenter %s:%d has invalid certificate", consenter.Host, consenter.Port)
```

Comment:
`errors.WithMessagef` would be a better choice, as the underlying error is generated by the fabric code.

Reply:
done